src/demo/OceanObject.js

```diff
@@ -44,20 +53,35 @@ export const loadAllFishPartImages = () => {
         fishPartImagesToLoad.push(partData);
       });
     });
+  return Promise.all(fishPartImagesToLoad.map(loadImage)).then(results => {
+    results.forEach(result => {
+      if (fishPartImages[result.data.partIndex] === undefined) {
+        fishPartImages[result.data.partIndex] = {};
+      }
+      fishPartImages[result.data.partIndex][result.data.variationIndex] =
+        result.img;
+    });
+  });
+};
 
-  return Promise.all(fishPartImagesToLoad.map(loadFishPartImage)).then(
-    results => {
-      results.forEach(result => {
-        if (fishPartImages[result.data.partIndex] === undefined) {
-          fishPartImages[result.data.partIndex] = {};
-        }
-        fishPartImages[result.data.partIndex][result.data.variationIndex] =
-          result.img;
-      });
-    }
-  );
+export const initMobilenet = () => {
```

Comment:
this method should probably go in `SimpleTrainer.js` or something like a `trainerHelpers.js` file

Reply:
I was moving all the mobilenet stuff out of SimpleTrainer to avoid the trainer needing to know any of the state.

I could move it into `trainerHelpers` though.

Reply:
Actually, it then saves to a variable in this file so that won't be very clean either. I'm going to leave it here for now and we can refactor it out later.